<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaListarMarca</class>
 <widget class="QDialog" name="TelaListarMarca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonVoltar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMarca">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>440</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ID">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>470</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblMarca">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>80</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_Marca">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>430</y>
     <width>63</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>Marca</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>480</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Alterar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>126</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alterar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
